_overviews/jdk-compatibility/overview.md

```diff
@@ -4,70 +4,79 @@ title: JDK Compatibility
 permalink: /overviews/jdk-compatibility/overview.html
 ---
 
-Scala runs primarily on the Java Virtual Machine (JVM).
+Scala's primary platform is the Java Virtual Machine (JVM). (Other supported platforms: [Scala.js](https://www.scala-js.org/), [Scala Native](https://scala-native.readthedocs.io/).)
 
-Sometimes new JVM versions require us to update Scala to remain compatible.
-
-And as Scala and the JVM improve independently over time, Scala may drop compatibility with older JVM versions, in order to better take advantage of new JVM features.
+Sometimes new JVM and JDK (Java Development Kit) versions require us to update Scala to remain compatible.
 
 ## Version compatibility table
 
-This table shows the first Scala release in each series that works with each JVM release.
-
-| JVM version | Minimum Scala versions                                                                        |
+| JDK version | Minimum Scala versions           | Recommended Scala versions                                 |
 |:-----------:|:----------------------------------------------------------------------------------------------|
```

Comment:
missing an update to this line to add another column to the header/content separator, which breaks the table.

```suggestion
|:-----------:|:---------------------------------|:-----------------------------------------------------------|
```